rel/app/windows/Livebook.cs

```diff
@@ -27,7 +27,12 @@ static void Main(string[] args)
             Application.SetCompatibleTextRenderingDefault(false);
             Application.Run(new LivebookApp(url));
 
-            var code = ElixirKit.API.Stop();
+            if (!ElixirKit.API.HasExited)
```

Comment:
Would it make sense to align the APIs to have is-running in both c# and swift?

Reply:
I think so but I wasn't sure. At the moment we have:

```swift
ElixirKit.API.start(terminationHandler:)
// because Process(terminationHandler:)

ElixirKit.API.isRunning
// because process.isRunning

ElixirKit.API.waitUntilExit
// because process.waitUntilExit
```

and

```csharp
ElixirKit.API.Start(exited: exited)
// because process.Exited += event

ElixirKit.API.HasExited()
// because process.HasExited

ElixirKit.API.WaitForExit()
// because process.WaitForExit
```

And I felt if I settled on one the others would find it confusing. Happy to unify though.

Reply:
I see, no strong opinion then. The API is small so making it more tailored for each platform is fine.